lib/TH/generic/simd/simd.h

```diff
@@ -9,7 +9,7 @@
 #endif
 
 // Can be found on Intel ISA Reference for CPUID
-#define CPUID_AVX2_BIT 0x10       // Bit 5 of EBX for EAX=0x7
+#define CPUID_AVX2_BIT 0x20       // Bit 5 of EBX for EAX=0x7
```

Comment:
But you want to shift `0x1` by 4, not by 5, right? (first bit + shift 4 = 5th bit).